src/careamics/careamist.py

```diff
@@ -668,6 +669,168 @@ def predict(
         )
         return convert_outputs(predictions, self.pred_datamodule.tiled)
 
+    def predict_to_disk(
+        self,
+        source: Union[PredictDataModule, Path, str],
+        *,
+        batch_size: int = 1,
+        tile_size: Optional[tuple[int, ...]] = None,
+        tile_overlap: Optional[tuple[int, ...]] = (48, 48),
+        axes: Optional[str] = None,
+        data_type: Optional[Literal["tiff", "custom"]] = None,
+        tta_transforms: bool = False,
+        dataloader_params: Optional[dict] = None,
+        read_source_func: Optional[Callable] = None,
+        extension_filter: str = "",
+        write_type: Literal["tiff", "custom"] = "tiff",
+        write_extension: Optional[str] = None,
+        write_func: Optional[WriteFunc] = None,
+        write_func_kwargs: Optional[dict[str, Any]] = None,
+        **kwargs,
+    ) -> None:
+        """
+        Make predictions on the provided data and save outputs to files.
+
+        The predictions will be saved in a new directory 'predictions' within the set
```

Comment:
From the user point of view, it might make more sense to have a mandatory `destination` folder.

I am not sure how that would fit API-wise with your future plans. If `destination` is a zarr, then it detect that in the future (together with the `write_type`?

Reply:
I think it can work if `destination` is a directory. I think it is better to not give too many options, makes the code less confusing

Reply:
Sounds like a good solution, `destination` as a folder.

Reply:
There is now `prediction_dir` parameter, default is "predictions". It can handle the directory being absolute or relative to the working directory set in CAREamist.